### PR TITLE
Feat : added P1 - Grid Component

### PR DESCRIPTION
**Pull request by TusharJade**
### added P1 - Grid Component

Screenshot (51)
Screenshot (50)

### CODE REVIEW

File: Components/Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title></title>
```
Comment (Avin008) [inline Components/Grid/grid.html]: add title

**Review by Avin008 — APPROVED** [review]
looks good to merge

File: Components/Grid/grid.html (hunk 1 above)
Comment (TusharJade) [reply Components/Grid/grid.html]: Will fix that now thanks

**Review by SumitSaundadkar — APPROVED** [review]
code is clean, try to use good class names, avoid spacing  other than that everything looks great. good to go for merge. code look great love the responsiveness of component library.

**Comment by TusharJade** [comment]
> code is clean, try to use good class names, avoid spacing other than that everything looks great. good to go for merge. code look great love the responsiveness of component library.

Thanks Sumit 🙌

**Pull request merged**